extensions/schema-compare/src/utils.ts

```diff
@@ -84,7 +85,7 @@ function connectionInfoToConnectionProfile(details: azdata.ConnectionInfo): azda
 	};
 }
 
-export async function verifyConnectionAndGetOwnerUri(endpoint: mssql.SchemaCompareEndpointInfo, caller: string, apiWrapper: ApiWrapper): Promise<string> {
+export async function verifyConnectionAndGetOwnerUri(endpoint: mssql.SchemaCompareEndpointInfo, caller: string, apiWrapper: ApiWrapper): Promise<string | undefined> {
```

Comment:
Would it be better to throw an error rather than have split logic above this for `undefined`?  No idea myself.

Reply:
Actually, the logic already exists to handle both the cases. I just realized that the last time when I worked on this file, I missed this change due to some merge conflicts, hence introduced this change back here.